etc/docker/kubernetes-kernel-tf-py-gpu/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM tensorflow/tensorflow:1.9.0-gpu-py3
+
+RUN pip install pycrypto
+
+ADD jupyter_enterprise_gateway*.tar.gz /usr/local/share/jupyter/kernels/
+
+COPY bootstrap-kernel.sh /etc/
+
+RUN addgroup --system --gid 620 eg-kernel && adduser --system --uid 620 --gid 620 eg-kernel && \
```

Comment:
is this a MUST have? Just want to make sure all our kernels have a standard configuration.

Reply:
As of #384 this _is_ the standard configuration for our k8s images.